.npmignore

```diff
@@ -0,0 +1 @@
+src/
```

Comment:
thoughts? It's not super relevant in this component, but we should do one pattern in all our components and converge on that...